getting-started/templates/GCP/Openshift/openshift_supplemental_values.yaml

```diff
@@ -0,0 +1,14 @@
+## Openshift prohibits running the pods as a root user. The redis and rabbitmq pods should be run as non-root users.
```

Comment:
Can we move `Openshift` folder parallel to `GCP` folder as Openshift is not specific to GCP?